src/main/java/login/UserManager.java

```diff
@@ -58,19 +72,14 @@ public void Login(String username, String password){
     }
 
 
-    // method to find the user in the allUsers and change the status to logged out
-    public void Logout(String username){
-        for(User person :AllUser.getAllUser()){
-            if(Objects.equals(person.getUsername(), username)){
-                person.setLoginStatus(false);
-            }else{LoginOutputBound.LogoutFail();}
-
-        }
-
+    /**
+     * To check if user already followed this other user
+     * and show error messages when the other user is already followed
+     * @param username string username of the user
+     * @param other string username of the other user
+     */
 
-    }
-
-    //Check if user already followed this other user(refactoring)
+    @Override
     public boolean CheckFollow(String username, String other){
```

Comment:
I think method names should be lowercase. To avoid problems, this can probably be fixed later on